packages/components/src/DatePicker/DatePicker.test.tsx

```diff
@@ -77,3 +72,48 @@ it("should not add the `react-datepicker-ignore-onclickoutside` when inline", ()
 
   expect(target).not.toHaveClass(className);
 });
+
+describe("Ensure ReactDatePicker CSS class names exists", () => {
+  it("should have the click outside class", async () => {
+    const { getByRole } = render(<ReactDatePicker onChange={jest.fn} />);
+    const input = getByRole("textbox");
+    const className = "react-datepicker-ignore-onclickoutside";
+
+    expect(input).not.toHaveClass(className);
+    await popperUpdate(() => fireEvent.focus(input));
+    expect(input).toHaveClass(className);
+  });
+
+  describe("Overwritten CSS class name", () => {
+    const classNames = [
+      ".react-datepicker",
+      ".react-datepicker__input-container",
+      ".react-datepicker-wrapper",
+      ".react-datepicker-popper",
+      ".react-datepicker__header",
+      ".react-datepicker__month",
+      ".react-datepicker__day",
+      ".react-datepicker__day--outside-month",
+      ".react-datepicker__day--selected",
+      ".react-datepicker__day-names",
+      ".react-datepicker__day-name",
+    ];
```

Comment:
Question: Just wondering if we do actually need to test these, or if we should just worry about just the ones that affect our system like the `react-datepicker-ignore-onclickoutside` class?

Reply:
@darryltec Woops my bad!  All of these are in the `DatePicker.css` file!  So we gotta check these exist